include/cuda/detail/__cuda_util

```diff
@@ -0,0 +1,41 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of libcu++, the C++ Standard Library for your entire system,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _LIBCUDACXX_CUDA_UTIL
+#define _LIBCUDACXX_CUDA_UTIL
+
+#include "../exception"
+#include "../std/detail/__config"
+#include "../std/detail/__pragma_push"
+#include "../std/version"
+
+#include <cuda_runtime_api.h>
+#include <string>
```

Comment:
We should be including and using our version of string, not the upstream standard library's string, right? @griwes?

Reply:
This is fine, we don't have cuda strings, and in a heterogeneous std universe this *would* be our string.